Lib/test/test_call.py

```diff
@@ -760,6 +760,32 @@ def __call__(self, *args):
                 self.assertEqual(expected, meth(*args1, **kwargs))
                 self.assertEqual(expected, wrapped(*args, **kwargs))
 
+    def test_setvectorcall(self):
+        from _testcapi import function_setvectorcall
+        def f(num): return num + 1
+        num = 10
+        self.assertEqual(11, f(num))
+        function_setvectorcall(f)
+        # make sure specializer is triggered by running > 50 times
+        for _ in range(51):
```

Comment:
Ditto about using `QUICKENING_WARMUP_DELAY` Maybe use `QUICKENING_WARMUP_DELAY * 5`

Reply:
used `*10` to be sure :)